CHANGELOG.rst

```diff
@@ -52,29 +52,6 @@ Protocol is now at version **1.12**. See `API changelog`_.
 Thanks to all contributors, with a special big-up for @gabisurita!
 
 
-4.3.4 (2016-11-17)
```

Comment:
I don't feel strongly about it if @Natim wants to keep them. My real concern is just that I don't understand the workflow for releases on a release branch.